### ISSUE TITLE
RLC-510WA wrong api calls for email switch

### ISSUE THREAD

**Issue by majkers** [fwestenberg/reolink#93]
I've noticed that there might be a problem in API calls for enabling email. For my camera (RLC-510 WA) with firmware v3.1.0.764_21121712 email switch has disappeared from Home Assistant integration https://github.com/fwestenberg/reolink_dev/issues/560 and I also have noticed that this API call

http://[CAMERA_IP]/api.cgi?cmd=getEmail&user=[USER]&password=[PASSWORD]

is not working and this one does

http://[CAMERA_IP]/api.cgi?cmd=getEmailV20&user=[USER]&password=[PASSWORD]

So in my opinion and according to this Relonk API guide:

https://drive.google.com/drive/folders/19vQBJia0wKvzwscA-EpTDSFV5OfNYUL6

for my camera command SetEmailV20 should be used

**Comment by pedro042** [comment]
It depends on firmware version; for example RLC-520A (older firmware) use cmd=getEmail and newer firmware use cmd=getEmailV20. But E1 Zoom use (latest firmware) use cmd=getEmail

**Comment by majkers** [comment]
Yeah ok but if you look at the source code e.g. ftp and some other features are used in both ways (with or without V20) depending on camera's firmware. That is not the case with email.
